### ISSUE TITLE
Parse error on Ubuntu 17.10

### ISSUE THREAD

**Issue by lambdageek** [lambdageek/Ward#1]
[Mono Jenkins](https://jenkins.mono-project.com/job/test-mono-mainline-staticanalysis/) upgraded to Ubuntu 17.10 (from 17.04).  The new glibc header broke `language-c` parsing:

```
CSourceUnitParseError /usr/include/x86_64-linux-gnu/bits/mathcalls-helper-functions.h:21: (column 40) [ERROR]  >>> Syntax Error !
  Syntax error !
  The symbol `__value' does not fit here.
```

Which is the same error as haskell/c2hs#192 - And the solution is the same - bump the `language-c` dependency to `0.7.1`

**Comment by lambdageek** [comment]
If I care about GCC < 7 (not sure), I may also be affected by visq/language-c#43 (taking `_Float128` as a `language-c` primitive is wrong in that case - instead `__float128` should be the primitive)

**Comment by lambdageek** [comment]
Fixed by 84b97a8
